src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -16,54 +15,107 @@ namespace Microsoft.FeatureManagement
     /// <summary>
     /// Used to evaluate whether a feature is enabled or disabled.
     /// </summary>
-    class FeatureManager : IFeatureManager, IDisposable
+    public sealed class FeatureManager : IFeatureManager
```

Comment:
Does this PR need to be merged/reviewed after the PR to make feature manager public is in?

Reply:
Yes.